docs/getting_started.rst

```diff
@@ -26,13 +44,23 @@ In ``fast_pauli``, we represent pauli matrices using the ``Pauli`` class. For ex
     pauli_y = fp.Pauli('Y')
     pauli_z = fp.Pauli('Z')
 
+    str(pauli_0)  # returns "I"
+
 We can also multiply two ``Pauli`` objects together to get a ``Pauli`` object representing the tensor product of the two pauli matrices.
+The result includes a phase factor because Pauli matrices do not necessarily commute. They exhibit the following `commutation relations <https://en.wikipedia.org/wiki/Pauli_matrices#Algebraic_properties>`_:
```

Comment:
So it isn't the commutation that is the problem (and two pauli matrices don't commute). It is the fact that the pauli matrices are not actually closed (two pauli matrices multiplying each other do not equal another pauli matrix).

The commutation doesn't actually come up in any of the stuff here. That would just be X @ Y != Y @ X, which we don't discuss